lib/language/messages_en.py

```diff
@@ -22,6 +22,7 @@ def all_messages():
             "subdomains": "find and scan subdomains",
             "Invalid_whatcms_api_key": "{0}",
             "searching_whatcms_database": "Searching for CMS on whatcms.org...",
+            "whatcms_monthly_quota_exceeded": "You have exceeded your monthly request quota",
```

Comment:
Please change to "You have exceeded your monthly WHATCMS request quota" - it is not clear which module the error message relates to